proto/osmosis/txfees/v1beta1/feetoken.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package osmosis.txfees.v1beta1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/x/txfees/types";
+
+message FeeToken {
+  option (gogoproto.equal) = true;
+
+  string denom = 1 [ (gogoproto.moretags) = "yaml:\"denom\"" ];
+  uint64 poolID = 2 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
```

Comment:
Would be good to clarified the use of poolID - if poolID is configured in a wrong way - what happens?